src/ODE.jl

```diff
@@ -66,10 +66,10 @@ function hinit(F, x0, t0, tend, p, reltol, abstol)
     # Returns first step, direction of integration and F evaluated at t0
     tdir = sign(tend-t0)
     tdir==0 && error("Zero time span")
-    tau = max(reltol*norm(x0, Inf), abstol)
-    d0 = norm(x0, Inf)/tau
+    tau = max(reltol.*abs(x0), abstol)
```

Comment:
This function is not specific to ode23s. This function has been written as per 'Solving Ordinary Differential Equations I' by Hairer and Wanner and they simply use the absolute values of each component. The 'tau' used in this code is simply the scaling factor 'sc' used in the book.